BUILD.adoc

```diff
@@ -221,6 +221,23 @@ SKIA_DIR=path SKIA_LIB_DIR=path cargo build --release --features "skia-backend"
 
 == macOS
 
+We assume that you have already built Skia itself.
+
+Install `harfbuzz` using your distributive's package manager.
+
+Using BrewInstall you can install it via:
+
+```
+brew install harfbuzz
+```
+
+`SKIA_DIR` should point to a Skia directory that contains the Skia `include` directory.
+`SKIA_LIB_DIR` should point to a Skia directory that contains `libskia.so`.
```

Comment:
`libskia.so` ->  `libskia.dylib`

Reply:
fixed, thanks

Reply:
`dylib`, not `a`.

Reply:
ah I was building skia as a static library
okay fixed now, thanks!